doc_build/BUILD

```diff
@@ -11,16 +11,39 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Generate the reference documentation.
+
+How to:
+   bazel build //doc_build:reference
+   cp bazel-bin/doc_build/reference.md docs/latest.md
+   git commit -m 'update docs' docs/latest.md
+"""
 
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+load("@rules_python//python:defs.bzl", "py_library")
 load("//:version.bzl", "version")
 
-# HOW TO:
-#   bazel build //doc_build:reference
-#   cp bazel-bin/doc_build/reference.md docs/latest.md
-#   git commit -m 'update docs' docs/latest.md
+filegroup(
```

Comment:
We should probably be using `pkg_files`es for these at some point.  It's kinda ironic that we aren't.

Reply:
> Might be worth saying that in the source file in question, but otherwise should be fine for our immediate purposes.

The PR description does say "crude, but mostly works".
My actual goal is not to make this code good. It is to provide salient examples of what Stardoc should do and get the functionality built on that side. Then everyone benefits.

Reply:
filegroup to pkg_files for this.
Yes. You have a good point. But
- another PR, because we would want to do it everywhere
- the entire 'srcs', recursive idiom is an ugly wart on this. Sometimes, you want rules to just look at your source repo overall, and not at package boundaries
- bazel is getting new native rules to help gather subdirectories exactly for this use case, so let's wait until those roll out.